src/DonationForms/resources/app/utilities/convertValuesToFormData.ts

```diff
@@ -3,9 +3,16 @@
  */
 export default function convertValuesToFormData(values: object): FormData {
     const formData = new FormData();
+    for (const valueKey in values) {
+        const value = values[valueKey];
 
-    for (const key in values) {
-        formData.append(key, values[key]);
+        if (value !== null && value === 'object') {
```

Comment:
Missing `typeof` here right?

```suggestion
        if (value !== null && typeof value === 'object') {
```